add-on/panels/panel.html

```diff
@@ -23,7 +23,7 @@
 
     <script type="text/javascript" src="shared/js/loopapi-client.js"></script>
     <script type="text/javascript" src="shared/js/utils.js"></script>
-    <script type="text/javascript" src="shared/js/models.js"></script>
+    <script type="text/javascript" src="add-on/panels/js/models.js"></script>
```

Comment:
This needs to be just `panels/js/models.js` (see the paths for some other panels files further down in the file).